deploy/charts/cert-manager/templates/deployment.yaml

```diff
@@ -158,6 +161,19 @@ spec:
           resources:
             {{- toYaml . | nindent 12 }}
           {{- end }}
+          # LivenessProbe settings are based on those used for the Kubernetes
+          # controller-manager. See:
+          # https://github.com/kubernetes/kubernetes/blob/806b30170c61a38fedd54cc9ede4cd6275a1ad3b/cmd/kubeadm/app/util/staticpod/utils.go#L241-L245
+          livenessProbe:
+            httpGet:
+              port: http-healthz
+              path: /livez
+              scheme: HTTP
+            initialDelaySeconds: 10
+            periodSeconds: 10
+            timeoutSeconds: 15
+            successThreshold: 1
+            failureThreshold: 8
```

Comment:
These values are configurable for the webhook, maybe we want to do the same thing here?
https://github.com/cert-manager/cert-manager/blob/a33a97f7a7299f3c505c3cf2ec2d8e3380ce1c28/deploy/charts/cert-manager/values.yaml#L336-L341